specification.md

```diff
@@ -749,7 +749,7 @@ Constructs an empty `JobSpec`.
 
 <div class="lang-bindings">
 
-__Python__:
+__Python__
```

Comment:
Unfortunately, these are part of a scheme that renders such choices into a nice tabbed format (see https://exaworks.org/job-api-spec/specification.html#jobexecutor) when served, as intended, through github-pages with some extra javascript libraries, but also looks somewhat reasonable when rendered as plain markdown. For the scheme to work, the colon is necessary.